src/shell/lua.d/openmpi.lua

```diff
@@ -8,8 +8,15 @@
 -- SPDX-License-Identifier: LGPL-3.0
 -------------------------------------------------------------
 
-shell.setenv ("OMPI_MCA_pmix", "flux")
-shell.setenv ("OMPI_MCA_schizo", "flux")
+local mpi, version = shell.getopt_with_version ("mpi")
+
+if mpi == "openmpi" and version == "5" then
+    shell.setenv ("OMPI_MCA_pmix", "^flux")
+    shell.setenv ("OMPI_MCA_schizo", "^flux")
+else
+    shell.setenv ("OMPI_MCA_pmix", "flux")
+    shell.setenv ("OMPI_MCA_schizo", "flux")
+end
```

Comment:
Do you know if this env var has to be set to `^flux` explicitly for openmpi v5+, or can it be set to nothing at all?

Reply:
Also, should the test be for `version >= 5`?

Reply:
> Do you know if this env var has to be set to ^flux explicitly for openmpi v5+, or can it be set to nothing at all?

pmix v5 doesn't have those plugins, but I thought maybe it would be OK to have "try this option" in our back pocket for opempi  v4 when it's not working.

> Also, should the test be for version >= 5?

OK, that makes sense.  The flux mca plugins aren't coming back, I'm sure!